moped-editor/src/components/GridTable/Filters.js

```diff
@@ -96,107 +96,6 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-/**
- * Generates a copy of an empty field
- * @param uuid
- * @return {Object}
- */
-const generateEmptyField = (uuid) => {
-  /**
-   * The default structure of an empty field
-   * @type {Object}
-   * @property {string} id - The uuid of the field
-   * @property {string} field - The name of the column
-   * @property {operator} operator - The name of the operator
-   * @property {string[]} availableOperators - A string array containing the names of available operators
-   * @property {string} gqlOperator - A string containing the GraphQL operator
-   * @property {string} envelope - The a pattern to use as an envelope
-   * @property {string} value - The text value to be searched
-   * @property {string} type - The type of field it is (string, number, etc.)
-   * @constant
-   * @default
-   */
-  const defaultNewFieldState = {
-    id: null,
-    field: null,
-    operator: null,
-    availableOperators: [],
-    gqlOperator: null,
-    envelope: null,
-    placeholder: null,
-    value: null,
-    type: null,
-    specialNullValue: null,
-    label: null,
-  };
-  return { ...defaultNewFieldState, id: uuid };
-};
-
-/**
- * Get configuration about the filters using the query parameters present in the URL
- * @param {Array} filters - filters from the URL {field, operator, value}
- * @param filtersConfig - The configuration object for the filters
- * @return {Object}
- */
-const makeInitialFilterParameters = (filters, filtersConfig) => {
```

Comment:
We don't need this helper anymore. It took the three values stored in the search parameters (field, operator, and value) and built up the shape of data that the UI expected from our predefined `FiltersCommonOperators` and `PROJECT_LIST_VIEW_FILTERS_CONFIG`. It was a bridge to the UI changes made in this PR. Most of this work happens in the render now.